iothub_client/samples/iothub_convenience_sample/iothub_convenience_sample.c

```diff
@@ -240,9 +241,14 @@ int main(void)
 
         // Set any option that are neccessary.
         // For available options please see the iothub_sdk_options.md documentation
+#ifndef SAMPLE_HTTP
+        bool traceOn = true;
+        (void)IoTHubDeviceClient_SetOption(device_handle, OPTION_LOG_TRACE, &traceOn);
+#endif
+        // ms_delay can be between 1 and 100 milliseconds
+        tickcounter_ms_t ms_delay = 10;
+        (void)IoTHubDeviceClient_SetOption(device_handle, OPTION_DO_WORK_FREQUENCY_IN_MS, &ms_delay);
```

Comment:
thanks for adding reference to the github issue for this change but also mention may be one or two lines to describe what problem are you solving here?